tailwind.config.js

```diff
@@ -0,0 +1,24 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{html,js}"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
+
+module.exports = {
+
+  plugins: [
+      require('flowbite/plugin')
+  ]
+
+}
+
+module.exports = {
+
+  content: [
```

Comment:
This configuration is meant for flowbite but you used only tailwind cdn so its irrelevant to this project, you will only need this when you do a cli configuration.